docs/development_suite/api-console/api-design/merge_collaboration.md

```diff
@@ -5,12 +5,17 @@ sidebar_label: Merge of configurations
 ---
 
 :::caution
-Currently, in the merge process, the following Design sections are not supported:
+This feature is not a `git merge`, which means it does not include all the content of the project repository during the comparison between the two branches.
```

Comment:
While this content is true, I'd not put this in such a negative way; i'd explain the concept by using more positive sentences.

Such as: 

- You can merge data from the following sections (with a note on advanced being available only when using enhanced workflow)
- These sections, instead are not supported ..., this means they won't be featured in the merge process and must be manually managed
- all configuration file that are manually managed within the repo are to be manually merged since they are not supported as well

Reply:
The gist here is to inform the user about the limitations of this feature, since the content subject of this PR is included in the "caution" box.

But I can rephrase the entire paragraph to explain clearly in positive terms what the merge can do and add the caution box only for this issue caused by the risk of losing configurations not supported. 

I'll update this comment once I'm done

Reply:
In a second thought, I've decided to apply your suggestion in the same caution box to avoid rewriting different paragraphs of the document. Let me know if you would prefer a different approach or if you encounter any mistake/typo.

Reply:
I agree with your consideration!